harvard-lts/CURIOSity
# Fix yarn error

**Fix the yarn errors and add a comma to results numbers**
* * *
)

# What does this Pull Request do?
A package upgraded that required us to alter some yarn build things, Doug's PR for the comma fix is also here since main was broken.

# How should this be tested?

Spin up local from this branch and this should build successfully (before it failed).

If you want to look at Doug's comma PR (I checked and merged already), hit search on any collection with 1000+ items

![screen_shot_2023-06-16_at_2 36 44_pm_360](https://github.com/harvard-lts/CURIOSity/assets/61880785/424b258f-d7e6-4d94-a1bf-c58541f9d61f)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties

Tag (@ mention) interested parties
